### PR TITLE
replace react quill with tiptap editor

### PR DESCRIPTION
**Pull request by thomastepi**
replaced quill editor with more stable and well maintained tiptap

### CODE REVIEW

File: frontend/src/components/Button/Button.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ Button.propTypes = {
   variant: PropTypes.oneOf(["text", "outlined", "contained"]),
   className: PropTypes.string,
   disabled: PropTypes.bool,
-  buttonType: PropTypes.oneOf(['primary', 'secondary-grey', 'secondary-purple', 'error'])
+  buttonType: PropTypes.oneOf(['primary', 'secondary', 'secondary-grey', 'secondary-purple', 'error'])
```
Comment (erenfn) [inline frontend/src/components/Button/Button.jsx:29]: Is this a new button style that wasn't present before?

**Review by uparkalau — APPROVED** [review]

**Review by uparkalau — DISMISSED** [review]

File: frontend/src/components/Button/Button.jsx (hunk 1 above)
Comment (uparkalau) [reply frontend/src/components/Button/Button.jsx:29]: @thomastepi please double check button options

File: frontend/package.json [hunk 2]
```diff
@@ -10,6 +10,14 @@
     "@mui/material": "^5.16.5",
     "@mui/x-date-pickers": "^7.3.1",
     "@mui/x-date-pickers-pro": "^7.3.1",
+    "@testing-library/jest-dom": "^5.17.0",
```
Comment (uparkalau) [inline frontend/package.json:13]: please delete @testing-library/jest-dom".
We do not need this.

**Review by erenfn — CHANGES_REQUESTED** [review]
Take a look at Vlad's comments

File: frontend/src/components/Button/Button.jsx (hunk 1 above)
Comment (thomastepi) [reply frontend/src/components/Button/Button.jsx:29]: i needed to use this button-disable style present in the button css file: /* .secondary-disabled */
.secondary.MuiButton-root:disabled {
  color: grey;
  background-color: white;
  border-color: #80808033;
  font-weight: 600;
}

**Review by erenfn — APPROVED** [review]

**Pull request merged**